package.json

```diff
@@ -280,7 +278,8 @@
     "webpack-bundle-analyzer": "4.7.0",
     "webpack-cli": "^4.2.0",
     "webpack-dev-server": "3.11.3",
-    "webpack-merge": "=4.1.5"
+    "webpack-merge": "=4.1.5",
+    "yarn": "1.22.19"
```

Comment:
Any special reason for why need to add yarn here? I have never added it when used yarn in the past, but maybe i missed something 🤔

Reply:
if we do not have yarn installed gloablly and if we have npx then we can do `npx yarn` and use the locally installed yarn

Reply:
Aha, I didnt know yarn could work like that 👍